components/playground/main.go

```diff
@@ -30,6 +30,8 @@ import (
 	"syscall"
 	"time"
 
+	"github.com/spf13/pflag"
+
```

Comment:
remove this empty line and `make fmt` again

Reply:
done